### PR TITLE
modify paddlenlp.ext_op to paddlenlp.op for docs

### PR DESCRIPTION
**Pull request by yingyibiao**
### PR types

Bug fixes
### PR changes

Docs

### CODE REVIEW

**Review by yingyibiao — COMMENTED** [review]
LGTM

File: docs/index.rst [hunk 1]
```diff
@@ -80,9 +80,9 @@
    paddlenlp.data <source/paddlenlp.data>
    paddlenlp.datasets <source/paddlenlp.datasets>
    paddlenlp.embeddings <source/paddlenlp.embeddings>
-   paddlenlp.ext_op <source/paddlenlp.ext_op>
    paddlenlp.layers <source/paddlenlp.layers>
    paddlenlp.metrics <source/paddlenlp.metrics>
+   paddlenlp.op <source/paddlenlp.op>
```
Comment (ZeyuChen) [inline docs/index.rst]: It should be `ops`

**Review by ZeyuChen — COMMENTED** [review]
Modify to `ops`

File: docs/index.rst (hunk 1 above)
Comment (yingyibiao) [reply docs/index.rst]: done

**Review by chenxiaozeng — APPROVED** [review]

**Pull request merged**